server/serverRoutes/reviews.js

```diff
@@ -40,5 +41,19 @@ module.exports = {
         res.status(200).send(data.data);
       })
       .catch(err => console.log('resultErr', err));
+  },
+  reviewsAdd: (req, res) => {
+    axios(requestConfig('post', api + 'reviews', req.body))
+      .then((data)=> {
+        res.status(200).send(data.data);
+      })
+      .catch(err => console.log('resultErr', err));
+  },
+  reviewsInteraction: (req, res) => {
```

Comment:
mentioned already - but sometime in future combining click objects and sending them all at once could be more efficient then each of us doing it individually